spec/features/bulkrax_export_spec.rb

```diff
@@ -63,15 +69,7 @@
   end
 
   describe "round-tripping" do
-    let(:exporter) do
-      create(:bulkrax_exporter_worktype,
```

Comment:
Was this not required?

Reply:
There were several definitions for this, which varied only in their export_source, so I put the definition up top, and had individual test set their export source to make it more dry.